### ISSUE TITLE
Allow changing username, password and update interval via UI

### ISSUE THREAD

**Issue by acidicX** [mitch-dc/volkswagen_we_connect_id#267]
**Is your feature request related to a problem? Please describe.**
As the update interval limit changes from time to time, I either have to edit the source code or remove all entities and use the setup again. Both options are not very user friendly, nor quickly done.

**Describe the solution you'd like**
Usually other integrations have a "config" button next to the entities, but VW We Connect ID has not. It would be ideal if that just displays the dialogue used for the initial setup, where you can change the three values again.

**Describe alternatives you've considered**
None

**Additional context**

Example with button

![grafik](https://github.com/user-attachments/assets/aa4768bd-22e5-4242-94b9-f580a95cc7f3)

vs

![grafik](https://github.com/user-attachments/assets/4c1033c3-1929-47bc-a130-1e46f4b6ece5)

---

Thanks for all the work so far! Really cool integration, usually works better and faster than the official app ;-)

**Comment by pdcastro** [comment]
> _[...] or remove all entities and use the setup again._

I agree that it would be a worthwhile user-friendliness improvement. 👍 But commenting on this point specifically, deleting and re-adding the integration (as in these [screenshots](https://github.com/mitch-dc/volkswagen_we_connect_id/pull/247#issuecomment-1936707669)) does not involve removing entities _individually,_ so it’s not as bad as it may sound. As far as I recall, no changes are needed to automations or scripts or other integrations that use the removed and re-added entities (e.g. schedulers), though restarting Home Assistant might be required in case those automations or scripts or other integrations do not recover by themselves.

**Comment by acidicX** [comment]
@pdcastro that is true, however only if you name the new/recreated entities *exactly* as before (notation could also change between versions), so there is quite some potential to go wrong. By "remove all entities" I meant what is done in the screenshots you linked.
